app/models/repository.rb

```diff
@@ -76,4 +77,13 @@ def update_pull_requests(only_open: false)
 
     pull_requests.count
   end
+
+  def update_forge_module
+    # we should save the json in Redis and cache it
+    content = Github.client.contents("voxpupuli/#{name}", path: 'metadata.json', query: { ref: 'master' })
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is too long. [106/100]](https://app.codacy.com/manual/VoxPupuli/vox-pupuli-tasks/pullRequest?prid=3891233)